lapce-data/src/editor.rs

```diff
@@ -1269,7 +1269,13 @@ impl LapceEditorBufferData {
             }
         }
 
-        self.update_completion(ctx, false);
+        match cmd {
+            EditCommand::InsertNewLine
```

Comment:
Recommend: encapsulating expression of whether an EditCommand is one that inserts a whitespace character.
Something like
```
impl EditCommand {
    fn is_insert_whitespace(&self) _> bool {
        use EditCommand::*;
        match self {
            InsertNewLine
            | InsertTab
            | NewLineAbove
            | NewLineBelow => true,
            _ => false
        }
    }
}
```
Note: I did not check if this compiles.

Rationale: If anything else needs to similarly adapt to whitespace, it'll be handy to expect they have the same behavior. Makes it look more similar to its counterpart in expressing "if not whitespace, then update_completion", which seems like a requirement to check beforehand for 2/3 calls to update_completion.

Significance: Pedantic. I'd recommend accepting this Pull Request with or without this suggestion.